dpgen/generator/run.py

```diff
@@ -3243,6 +3243,9 @@ def post_fp_vasp (iter_index,
                         assert(use_ele_temp)
                         ele_temp = job_data['ele_temp']
                         all_te.append(ele_temp)
+            elif len(_sys) >= 2:
+                raise RuntimeError("The vasp parameter NSW should be set as 1") 
+                icount+=1
```

Comment:
Line 3248 will not be executed when line 3247 raises an error. So I don't know the role of line 3248.

Reply:
I misunderstand the function RuntimeError(). The line 3248 should be removed.